features/step_definitions/view_restaurant_page_steps.rb

```diff
@@ -1,6 +1,8 @@
 Given("the following restaurants exist") do |table|
   table.hashes.each do |hash|
-    FactoryGirl.create(:restaurant, hash)
+    restaurant = FactoryGirl.create(:restaurant, hash.except(:restaurant_category))
```

Comment:
hash.except[:restaurant_category]  gives me ArgumentError: Trait not registered: Thai
hash.except(:restaurant_category)  gives me AssociationTypeMismatch:
When I run hash.except[:restaurant_category] in pry I get "Thai"

Reply:
Try it with a bang. !  Also, check Ruby documentation for excluding a key from hash.